tests/jquery.flot.touchNavigate.Test.js

```diff
@@ -67,7 +67,50 @@ describe("flot touch navigate plugin", function () {
             expect(xaxis.max).toBeCloseTo(initialXmax - (initialXmax - midPointCoords.x) * (1 - 1/amount), 6);
             expect(yaxis.min).toBeCloseTo((midPointCoords.y - initialYmin) * (1 - 1/amount) + initialYmin, 6);
             expect(yaxis.max).toBeCloseTo(initialYmax - (initialYmax - midPointCoords.y) * (1 - 1/amount), 6);
-          });
+        });
+
+        it('for multiple touches should take into account just the first two of them and act acordingly',function() {
```

Comment:
Dont' forget to add a test for the case when everything starts correctly with 2 touches but a 3rd one interfere.